### PR TITLE
Fixed typo issues and make the doc clear to understand

### PR DESCRIPTION
**Pull request by nanjj**

### CODE REVIEW

**Comment by gcolliso** [comment]
@jnmoyne could you review? I believe @matthiashanel is out this week but if we need to wait for him to return that is fine as well.

**Comment by jnmoyne** [comment]
Will take a look, but it will take a few days before I can get to it

**Comment by jnmoyne** [comment]
@matthiashanel most of the changes are indeed how to install server, nats, nsc, etc... re-formatting of some sections (e.g. changing . to : at the end of some lines) and a little bit of added content around the 'resolver' functionality. 

e.g. this extract below:
```
If your setup has few Accounts, and the Users in the accounts not
 changed offen, and/or you are comfortable reloading server configs
 when accounts/users changed, then save yourself from the complexity
 and do not use JWT. Regular static configuration -- possibly with
 NKEYs -- will work just fine for you.

 If your accounts are not changed offen, and you have a lot of users in
 the accounts, and your `nats-server` cluster is not huge(say 3 or 5
 cluster members), JWT `memory-resolver` is your choice.

 If your accounts are huge and changed dynamic, and your nats cluster
 topology not a simple one, and can not accept so many servers'
 reloading, JWT `nats-resolver` is recommended.
```

... which I am ok with although personally I would maybe put forward the built-in resolver as the default resolver to use with JWT, and memory resolver as an option if you don't want administrators to be able to make change to accounts using NSC...

But overall it's all LTGM and thanks for the contribution.

(@matthiashanel the best way IMHO to review the changes is to click on the commit from this page and then hit the '.' key to switch to the built-in "IDE")

File: using-nats/developing-with-nats/tutorials/jwt.md [hunk 1]
```diff
@@ -318,78 +352,144 @@ In a large organization the centralized configuration approach can lead to less
...
+  been dedicated to the decentralized nature of NATS JWT usage.
 
 ### NATS JWT Hierarchy
 
 #### Decentralized Chain of Trust
 
-A `nats-server` is configured to trust an operator. Meaning, the Operator JWT is part of its server configuration and requires a restart or `nats-server --signal reload` once changed. It is also configured with a way to obtain account JWT in one of three ways (explained below).
+A `nats-server` is configured to trust an operator. Meaning, the
+Operator JWT is part of its server configuration and requires a
+restart or `nats-server --signal reload` once the configration
```
Comment (matthiashanel) [inline using-nats/developing-with-nats/tutorials/jwt.md:387]: configuration

**Review by matthiashanel — COMMENTED** [review]
basically LGTM, but found one typo

**Comment by matthiashanel** [comment]
> ... which I am ok with although personally I would maybe put forward the built-in resolver as the default resolver to use with JWT, and memory resolver as an option if you don't want administrators to be able to make change to accounts using NSC...

I'd be ok with that, got a particular phrasing in mind?

**Comment by matthiashanel** [comment]
@nanjj are there any significant changes since your last push? can't diff from my last review.

**Review by matthiashanel — COMMENTED** [review]
Found the change, makes it a lot clearer. just a few comments.

File: using-nats/developing-with-nats/tutorials/jwt.md [hunk 2]
```diff
@@ -318,78 +352,154 @@ In a large organization the centralized configuration approach can lead to less
...
+    eventually converge on the union of all account JWTs known to
+    every participating `nats-server`,
+  * Requires persistent storage in the form of a NON-NTFS directory
+    for `nats-server` to exclusively write into,
+  * Optionally, directly supports Account JWT removal,
+  * Between `nats-resolver` and `url-resolver`, the `nats-resolver` is
+    the clear recommendation.
+
+If your setup has few accounts, say, 1 to 3 acounts in total, and the
+users in the accounts are few(say, less 10), and not changed offen,
```
[the remark this answers is not in the document]
Comment (nanjj) [reply using-nats/developing-with-nats/tutorials/jwt.md]: Finally I removed these noises and make JWT `nats-resolver` as the only recommendation.

**Review by matthiashanel — APPROVED** [review]
LGTM

**Comment by matthiashanel** [comment]
@nanjj thanks a lot for your contribution. Glad you found that document useful enough to contribute to it.

**Pull request merged**